nemo/collections/asr/data/audio_to_label.py

```diff
@@ -162,10 +165,10 @@ def sliced_seq_collate_fn(self, batch):
                 assumes the signals are 1d torch tensors (i.e. mono audio).
             fixed_length (Optional[int]): length of input signal to be considered
         """
-        slice_length = self.featurizer.sample_rate * self.time_length
+        slice_length = int(self.featurizer.sample_rate * self.time_length)
         _, audio_lengths, _, tokens_lengths = zip(*batch)
         slice_length = min(slice_length, max(audio_lengths))
-        shift = 1 * self.featurizer.sample_rate
+        shift = int(self.featurizer.sample_rate * self.shift_length)
```

Comment:
Hi @nithinraok  Could you have a look at it here? Whether it will impact your models? Thanks!

Reply:
No, this is good. Thanks :)

Reply:
Awesome! Thanks!